build-ffmpeg

```diff
@@ -679,9 +678,43 @@ download "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.bz2"
 	--extra-libs="${EXTRALIBS}" \
 	--pkgconfigdir="$WORKSPACE/lib/pkgconfig" \
 	--pkg-config-flags="--static" \
-	--prefix="${WORKSPACE}"
+	--prefix="${WORKSPACE}" \
+	--disable-stripping \
+	--enable-debug
 
 execute make -j $MJOBS
+
+if is_mac; then
+	execute dsymutil libavcodec/libavcodec.dylib -o libavcodec.dSYM
+	execute dsymutil libavdevice/libavdevice.dylib -o libavdevice.dSYM
+	execute dsymutil libavfilter/libavfilter.dylib -o libavfilter.dSYM
+	execute dsymutil libavformat/libavformat.dylib -o libavformat.dSYM
+	# we're not building this one
+	# execute dsymutil libavresample/libavresample.dylib -o libavresample.dSYM
```

Comment:
don't we need libavresample to resample things? or is it bundled into something else?

Reply:
https://github.com/performous/performous/issues/396

No, it uses `libswresample`

Reply:
ah, maybe add that to the comment?